e2e/src/pages/app.po.ts

```diff
@@ -41,7 +38,7 @@ export class AppPage {
         this.clickElement(by.css('label[for="download-btn-toggle"]')),
         this.clickElement(by.css('label[for="navigate-btn-toggle"]')),
         this.clickElement(by.css('label[for="print-btn-toggle"]')),
-        this.clickElement(by.css('label[for="rotate-btn-toggle"]')),
+        // this.clickElement(by.css('label[for="rotate-btn-toggle"]')),
```

Comment:
why not remove these comments completely?

Reply:
Will take care that in code optimisation